test/Microsoft.AspNetCore.Watch.BrowserRefresh.Tests/BlazorWasmHotReloadMiddlewareTest.cs

```diff
@@ -41,7 +41,7 @@ public async Task DeltasAreSavedOnPost()
 
             // Assert
             AssertDeltas(deltas, middleware.Deltas);
-            Assert.NotEmpty(context.Response.Headers["ETag"]);
+            Assert.NotEqual(0, context.Response.Headers["ETag"].Count);
```

Comment:
That seems... odd. Was `NotEmpty` not working or was it an analyzer complaining about it?